README.md

```diff
@@ -80,6 +80,7 @@ req.on('error', function(err) {
 options = {}
     // Set to true to have any gzip stream automatically decompressed before saving
     options.gunzip = false;
+    options.headers = {'User-Agent': 'Node'};
```

Comment:
I added these default headers, I also enabled users to set options.headers and passed it into request.